packages/compass-data-modeling/src/store/generate-diagram-wizard.ts

```diff
@@ -414,8 +448,15 @@ export function confirmSelectDatabase(): DataModelingThunkAction<
           }),
       });
     } catch (err) {
```

Comment:
I only just remembered we don't surface these errors when listing collections and databases.
https://jira.mongodb.org/browse/COMPASS-5275 
So this won't throw. We might want to look at doing this differently. Maybe it's time for that tech debt. 
Let's sync on this?